cpp/src/community/edge_triangle_count_impl.cuh

```diff
@@ -136,7 +136,7 @@ edge_property_t<graph_view_t<vertex_t, edge_t, false, multi_gpu>, edge_t> edge_t
   auto edge_first = thrust::make_zip_iterator(edgelist_srcs.begin(), edgelist_dsts.begin());
 
   size_t edges_to_intersect_per_iteration =
-    static_cast<size_t>(handle.get_device_properties().multiProcessorCount) * (1 << 17);
+    static_cast<size_t>(handle.get_device_properties().multiProcessorCount) * (1 << 13);
```

Comment:
Is there a reason for 13?  Should this be some sort of named constant so it's easier to change?

Reply:
It is a good idea to expose it in order to easily control the chunk size. I temporarily set a lower value so that I can process a larger graph when running at scale